math/pi/spigotpi.go

```diff
@@ -0,0 +1,57 @@
+// spigotpi.go
+// description: A Spigot Algorithm for the Digits of Pi
+// details:
+// implementation of Spigot Algorithm for the Digits of Pi - [Spigot algorithm](https://en.wikipedia.org/wiki/Spigot_algorithm)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see spigotpi_test.go
+
+package pi
+
+import "strconv"
+
+func PiSpigot(n int) string {
+	pi := ""
+	boxes := n * 10 / 3
+	reminders := make([]int, boxes)
+	for i := 0; i < boxes; i++ {
+		reminders[i] = 2
+	}
+	heldDigits := 0
```

Comment:
`digitsHeld` instead of `heldDigits`
```suggestion
	digitsHeld := 0
```

Reply:
Done

Reply:
is it resolved? - I don't see it outdated. I don't think you have changed this variable - you might have missed it, haha 😄